Assets/Scripts/Manager/FinalSceneMusicManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class FinalSceneMusicManager : MusicManager{
```

Comment:
yes, I don't want to crowd music manager, final scene is the only scene where you change the music a lot, and this file is used to track all the music needed for final scene, and I can just play them by index